HealthMonitoring.Forwarders/IEndpointMetricsForwarder.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace HealthMonitoring.Forwarders
+{
+    public interface IEndpointMetricsForwarder
+    {
+        string Name { get; }

```

Comment:
This could be checked then on the class type assuming that only one instance of given class could be added.